x/incentives/keeper/gauge_test.go

```diff
@@ -585,6 +585,9 @@ func (s *KeeperTestSuite) TestCreateGauge_NoLockGauges() {
 }
 
 func (s *KeeperTestSuite) TestCreateGroupGauge() {
+	// TODO: Re-enable this once gauge creation refactor is complete in https://github.com/osmosis-labs/osmosis/issues/6404
+	s.T().Skip()
```

Comment:
Could we explicitly track each test skip TODO as acceptance criteria of #6404?

Reply:
Added